### PR TITLE
Update clowdapp.yml

### PR DESCRIPTION
**Pull request by prichard77**
Added a spec, testing section to support smoke tests.

### CODE REVIEW

**Review by rodrigonull — APPROVED** [review]

**Comment by prichard77** [comment]
What do I need to do to get this merged? Does it need another reviewer/approver? Any suggestions whom I should add as a reviewer?

**Comment by dehort** [comment]
@prichard77 you should add me as a reviewer.

**Review by dehort — APPROVED** [review]
I have no idea what that change does.  :)  So...LGTM

**Pull request merged**
